### ISSUE TITLE
Let's document: plan9

### ISSUE THREAD

**Issue by yutyo** [tldr-pages/tldr#5657]
[plan 9](https://p9f.org/) is not dead, its just resting.

**Comment by sbrl** [comment]
Thanks for requesting this set of pages! Do you want to have a go yourself?

Pages are best created by people who understand the command they are about.

If you do, take a quick read of [CONTRIBUTING.md](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md), which guides you through the process of creating a new page here. Then, just simply send a pull request to this repository - and if you need any assistance, just ask :-)

**Comment by Waples** [comment]
@yutyo do you know https://tools.suckless.org/9base/ ? ^^

**Comment by yutyo** [comment]
> @yutyo do you know https://tools.suckless.org/9base/ ? ^^

Yeah. Would you suggest to add 9base commands to common instead of opening a different page for plan9?

**Comment by yutyo** [comment]
It seems like some plan9 commands seem nearly identical to those that already exist in *nix systems, yet their capacity is different. So I am asking you:

Would you prefer a page for `plan9` or shall we document the commands in `commons`?

**Comment by bl-ue** [comment]
@yutyo I'd say put plan9-specific commands (not available on any other systems) in a new plan9 folder.  Otherwise, put them in common, but if they already exist there, do nothing.

**Comment by sbrl** [comment]
- If a command is specific to a single platform, then it belongs in its' own platform directory
 - If a command is identical across multiple platforms, then it belongs in `common`
 - If a command is identical across multiple platforms except 1, then put a version in `common` for the platforms that are identical, and another version in the specific platform folder for the platform where it is different.

**Comment by Managor** [comment]
What is the reason that there hasn't been any progress on this front? Are these pages not needed?

**Comment by spytheman** [comment]
The reason is that Plan 9 is dead.

**Comment by sbrl** [comment]
@Managor tldr-pages relies on contributions from the community (people like you!) to write and contribute pages.

Hence, progress on this issue is likely to be limited by the fact that there is limited interest from the community in contributing to this meta-issue at the moment.
